src/__tests__/Request.spec.ts

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { sendToTranslate } from "@/translation/request";
+
+const mockResponse = "привіт";
+
+jest.mock("axios", () => ({
+  get: jest.fn(() => Promise.resolve(mockResponse)),
+}));
+
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+const sourceLang = "en";
+const targetLang = "uk";
+
+it("mocking the axios call to get translation", async () => {
+  mockedAxios.get.mockResolvedValue({
+    data: mockResponse,
+  });
+  const result = await sendToTranslate(sourceLang, targetLang, "hi");
+  expect(mockedAxios.get).toHaveBeenCalledTimes(1);
+  setTimeout(() => {
+    expect(result).toBe(mockResponse);
+  }, 0);
+});
+it("called func returns error ", async () => {
```

Comment:
Could you test that `sendToTranslate` will throw an exception if 'axios' `get` method reject the promise? 
In the first test `sendToTranslate` function throws an exception too
Now you can change the mock response with `Promise.reject`
Place `expect(await sendToTranslate(sourceLang, targetLang, "hi")).toThrowError();` inside `try`. And ignore `catch`:

```
} catch (e) {
    // ignore failures
}
```